shoov/modules/custom/shoov_ui_build/drush/shoov_ui_build.drush.inc

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @file
+ * Drush commands for UI Builds.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function shoov_ui_build_drush_command() {
+  $items = array();
+
+  $items['delete-old-ui-builds'] = array(
+    'description' => 'Delete old UI builds and screenshots.',
+  );
+
+  return $items;
+}
+
+/**
+ * Command callback for "delete-old-ui-builds".
+ *
+ * Deletes ui-builds that are older than one month, meanwhile keeping 10 builds
+ * in each repository.
+ */
+function drush_shoov_ui_build_delete_old_ui_builds() {
+
+  $query = new EntityFieldQuery();
+  $result = $query
+    ->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'repository')
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->propertyOrderBy('nid', 'ASC')
+    ->execute();
+
+  if (empty($result['node'])) {
+    // Stop if there's not repositories.
+    drush_log('No reposirories found', 'warning');
+    return;
+  }
+
+  $ids = array_keys($result['node']);
+  $repos = node_load_multiple($ids);
+  foreach ($repos as $repo) {
+    drush_print('Checking Repository "' . $repo->title . '"');
+
+    // Need to use db_select() because field condition override in OG doesn't
+    // work in a Drush command.
+    $ui_build_query = db_select('og_membership', 'm');
```

Comment:
the only difference is that in devel_init() you were logged in, and in drush you are anon. You can simple set the user's account as the repo owner or use [this](https://github.com/shoov/shoov/commit/09aba93407fc766d7386cba938941f9e90ce55c1) tag